internal/runbits/refresh.go

```diff
@@ -27,7 +27,7 @@ func RefreshRuntime(auth *authentication.Auth, out output.Outputer, an analytics
 	}
 
 	if !changed && isCached {
-		out.Print(locale.Tl("pkg_already_uptodate", "Requested dependencies are already configured and installed."))
+		out.Notice(locale.Tl("pkg_already_uptodate", "Requested dependencies are already configured and installed."))
```

Comment:
`output.Notice()` is ignored by JSON/structured outputs. All of these types of informative messages need to be output as notices from now on, or they will contaminate JSON/structured output.